includes/WikiManager.php

```diff
@@ -207,6 +205,8 @@ public function rename( string $new ) {
 				]
 			);
 		}
+		
+		$this->recacheJson( $new );
```

Comment:
Should just be (). The original place is needed

Reply:
Thanks, done.